### DISCUSSION TITLE
Tuya TS011F_plug_3 ( _TZ3000_5f43h46b) not extending network & meshing with other TS011F_plug_3

### DISCUSSION THREAD

**Discussion by MaxDev98** [Koenkk/zigbee2mqtt#25309] [Devices]
Hey guys,

currently i'm using Z2M with the following setup:
Raspberry Pi 3B
SONOFF ZBDongle-P
Zigbee2MQTT version 1.42.0
Coordinator type zStack3x0
Coordinator revision 20240710 (was: 20210708 before, same issue)
Frontend version 0.7.6
zigbee-herdsman-converters version 20.58.0
zigbee-herdsman version 2.1.9

and 2 TS001F_plug_3 (_TZ3000_typdpbpg)
![Screenshot 2024-12-23 202615](https://github.com/user-attachments/assets/dcb32eb7-3217-4fc4-9909-980d59aa87e6)

Everything is working fine, both devices are in direct range of the coordinator. 
So to extend the network and monitor the energy of other devices, i bought another 10 Tuya smart plugs on AliExpress.

The new plugs are identified as Tuya TS011F_plug_3 (_TZ3000_5f43h46b)
![Screenshot 2024-12-23 202936](https://github.com/user-attachments/assets/fa8fefe7-1779-410a-b5a0-1df4a68e14d2)

Unfortionately the new plugs are only working then they can see the coordinator. If i'm trying to use one of the new plugs on a location that is not in range of the coordinator but is reachable by both old plugs (only one room away) the new plugs are unreachable. So it looks like they are not meshing/connecting with the old smart plugs, even though the old plugs are meshing to other devices (Hue lights) 2 rooms away.
(When the Hue lights are on, the new plugs will connect to them.. But since they wont have AC power all the time, thats no solution)

Scenarios:
1. Coordinator -> old plug: working
2. Coordinator -> new plug: working
3. Coordinator -> old plug -> Hue lights: working
4. Coordinator -> old plug -> new plug: not working
5. Coordinator -> old plug -> Hue lights -> new plug: working

e.g. when trying to set the LED status
```
error 2024-12-23 20:40:17z2m: Publish 'set' 'indicator_mode' to 'Steckdose Drucker' failed: 'Error: ZCL command 0x70b3d52b6011e6cc/1 genOnOff.write({"tuyaBacklightMode":3}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Timeout - 60540 - 1 - 138 - 6 - 4 after 10000ms)'
```

So i though maybe a firmware update may help, but for the old plugs it says that the Firmware is already up to date.
The new plugs wont respond to OTA Requests, even if in direct coordinator range:
```
info 2024-12-23 20:33:25z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/ota_update/check', payload '{"data":{"id":"Steckdose Drucker"},"error":"Failed to check if update available for 'Steckdose Drucker' (Device didn't respond to OTA request)","status":"error","transaction":"pjqdt-3"}'
error 2024-12-23 20:33:25z2m: Failed to check if update available for 'Steckdose Drucker' (Device didn't respond to OTA request)
```

Something that i noticed as well:
The old plugs are reporting energy changes every 1-5 seconds, the new plugs only every 60 seconds and changing the reporting interval won't apply to the new plug. Still reporting every 60 seconds.

Does anyone may have an idea and can help me how to troubleshoot and fix this issue?
I also ordered a temporary tuya bridge that will be delivered on Friday, so maybe i can update the firmware with this bridge.. 
(or is there maybe a chance to flash the plain tuya firmware on it?)

Thank you!
If you need more infos, please feel free to tell me!

Edit:
I updated the coordinator firmware to 20240710 without any luck/changes.

**Comment by SiggiBeu** [comment]
Hi,

did you use an external converter to add this plug to your Zigbee2MQTT network? 
I have I guess an identical plug but cannot get it working since Z2M reports: manufacturer unsupported.

Any info appreciated.
